engine/src/test/java/io/zeebe/engine/processing/timer/ConcurrentTimerEventTest.java

```diff
@@ -152,10 +152,6 @@ public void shouldRejectTriggerCommandIfElementInstanceIsLeft() {
             .workflowInstance(WorkflowInstanceIntent.ELEMENT_TERMINATING, eventActivated.getValue())
             .key(eventActivated.getKey())
             .causedBy(1),
-        RecordToWrite.event()
-            .workflowInstance(WorkflowInstanceIntent.ELEMENT_TERMINATED, eventActivated.getValue())
-            .key(eventActivated.getKey())
-            .causedBy(2),
```

Comment:
This test failed with the detection. Here it is missing the `CANCEL` command for the timer. But adding the command would lead to a different rejecting reason of the `TRIGGER` command (`NOT_FOUND` instead of `INVALID_STATE`). So, I removed the follow-up event `ELEMENT_TERMINATED`.